features/steps/test_starter/{{ cookiecutter.repo_name }}/src/{{ cookiecutter.python_package }}/settings.py

```diff
@@ -25,11 +25,13 @@
 # CONF_SOURCE = "conf"
 
 # Class that manages how configuration is loaded.
-# from kedro.config import TemplatedConfigLoader
-# CONFIG_LOADER_CLASS = TemplatedConfigLoader
+# CONFIG_LOADER_CLASS = ConfigLoader
 # Keyword arguments to pass to the `CONFIG_LOADER_CLASS` constructor.
 # CONFIG_LOADER_ARGS = {
-#     "globals_pattern": "*globals.yml",
+#       "config_patterns": {
+#           "spark" : ["spark*/"],
+#           "parameters": ["params*", "params*/**", "**/params*"],
+#       }
```

Comment:
What is this change for?

Reply:
Just to keep consistency between the `settings.py` file in the template and the `test_starter`